htlcswitch/link.go

```diff
@@ -653,6 +653,8 @@ func (l *channelLink) syncChanStates() error {
 
 	var msgsToReSend []lnwire.Message
 
+	time.Sleep(time.Second * 5)
```

Comment:
i've noticed this flake mostly on w/ the neutrino backend, do you know why it happens more so in that case? maybe it is possible w/ all backends but having a full node makes it less likely to crop up?